doc/radosgw/s3/commons.rst

```diff
@@ -20,10 +20,19 @@ The second method identifies the bucket via a virtual bucket host name. For exam
 
 To configure virtual hosted buckets, you can either set ``rgw_dns_name = cname.domain.com`` in ceph.conf, or add ``cname.domain.com`` to the list of ``hostnames`` in your zonegroup configuration. See `Ceph Object Gateway - Multisite Configuration`_ for more on zonegroups.
 
+.. prompt:: bash $
+	ceph config set client.rgw.<ceph authx client for rgw> rgw rgw_dns_name cname.domain.com
+
 .. tip:: We prefer the first method, because the second method requires expensive domain certification and DNS wild cards.
 
 .. tip:: You can define multiple hostname directly with the :confval:`rgw_dns_name` parameter.
 
+.. tip:: When SSL is enabled, the certificates need to use a wildcard in the domain name so it matches the bucket subdomains
+
+.. tip:: If all the rgw servers share same dns name, then you can use `ceph config set client.rgw` to set the dns name for all the rgw servers.
```

Comment:
I think this could be worded better, though I don't understand this functionality well enough to know for about "share DNS name".

I think something like

>.. tip:: If all RADOS GateWay (RGW) daemons share the same \`\`dns_name\'\', you can set \`\`dns_name\`\` for all of them at once by running \`\`ceph config set client.rgw\`\`.

Also, while you're editing this file, might I prevail upon you to s/hostname/hostnames/ two lines above?

Reply:
> I think this could be worded better, though I don't understand this functionality well enough to know for about "share DNS name".

radosgw instances would share a dns name if they're behind the same proxy. clients send a `Host` header with the proxy's dns name, and that's what radosgw matches against `rgw_dns_name`

this proxy configuration probably deserves its own note or tip